### PR TITLE
9 implement groups

### LINKED ISSUE #9: Implement groups
Investigate [PeerJS Group](https://github.com/ElizabethHudnott/peerjs-groups) and implement a room system so people can create party

### PR DESCRIPTION
**Pull request by codeeMadness**
Hi,

This has follow your suggestion to implement peerjs-group.
However, I've just finished the implementation of chat group.
We will need other tickets to handle spawn players in group and call video in group.

So, please help to merge this first.
Many thanks.

### CODE REVIEW

**Comment by Liquid-Blocks** [comment]
I have been experimenting a bit with this group feature and sadly do not think it is ready to be merged.

**Problem 1:** When creating a group, I am prompted with a message saying I need permission. 
image

**Problem 2:** There seems to be no ways of joining a group?
image

There is no hurry, please take the time to complete issues properly so we don't accumulate cleanup jobs.

**Comment by codeeMadness** [comment]
Hi @Liquid-Blocks ,

I've just updated the code for cover above issues.

https://user-images.githubusercontent.com/102911684/171687868-716079d7-6247-4ada-bccf-27aed5035b45.mp4

Please take a look on this video, feel free to add some feedbacks, then I'm willing to update more.

File: js/util/RemoteController.js [hunk 1]
```diff
@@ -2,13 +2,24 @@ import { Vector3 } from 'https://cdn.skypack.dev/pin/three@v0.137.0-X5O2PK3x44y1
...
 import { AvatarController } from '../entities/AvatarController.js';
 import { toggleCallBox, callBox, addCamera } from '../UiElements/callBox.js';
+import { PeerGroup } from './peerjs-groups.js';
+import alertBox from '../UiElements/alertBox.js';
 
 // PeerJs is injected in the window
 const Peer = window.Peer;
+const SIGNALLING_OPTIONS =  {
+    secure: 'wss://',
+    host: 'spacetime-peerserver.herokuapp.com',
```
Comment (Liquid-Blocks) [inline js/util/RemoteController.js:16]: Hi @codeeMadness could you please open a new pr to include the server side code into the repository.

This repository is public, be careful not to include any private key or token.

thank you

**Review by Liquid-Blocks — APPROVED** [review]
Thank you for the update!

**Pull request merged**